src/features/projects/components/PopupProject.tsx

```diff
@@ -115,7 +128,8 @@ export default function PopupProject({
                   id={`ProjPopDonate${project.id}`}
                   ref={buttonRef}
                   onClick={handleDonationOpen}
-                  className={'donateButton'}
+                  className={`donateButton ${donateButtonBackgroundColor}`}
+                  disabled={!project.allowDonations}
```

Comment:
I assume this must be the following as otherwise the button is still disabled - have you even tested this?

```suggestion
                  disabled={!project.properties.allowDonations}
```

Reply:
I'm sorry I forgot to check the functionality of buttons. I've made the changes.
Also, I noticed that we are not displaying the buttons for non donatable projects so I've simply removed all the conditions specific to button. Hence, there are no gray and disabled buttons.